public/css/stylesheet.css

```diff
@@ -0,0 +1,13 @@
+/*.low-usage {
```

Comment:
Consider removing stylesheet.css?

Reply:
Ooops - yep, I forgot I'd left that in there, good catch!